README.md

```diff
@@ -1,12 +1,14 @@
-# Rocky 2.0.2
+# Rocky 3.0.0 #
 
-Rocky is an framework for building powerful and scalable APIs for your imp-powered products. The Rocky library consists of the following classes:
+Rocky is an framework for building powerful and scalable APIs for your imp-powered products.
+
+**Important** From version 3.0.0, Rocky is implemented as a table rather than a class. **This is a breaking change**. This change has been made to reduce Rocky’s memory footprint and to ensure that Rocky is available solely as a singleton. For full details on updating your code, please see [**Rocky Usage**](#rocky-usage), below.
```

Comment:
This update actually increases Rocky's memory footprint. The adjustment to the table just ensures that the new features don't increase by much.